spec/imsc-hrm.html

```diff
@@ -539,10 +542,10 @@ <h2>General</h2>
       example, if the Back Buffer is copied twice to Front Buffer between two consecutive video frame boundaries of the
       <a>Related Video Object</a>.</p>
 
-      <p>It SHALL be an <a>error</a> for the Presentation Compositor to fail to complete painting pixels for
+      <p>It is an <a>error</a> for the Presentation Compositor to fail to complete painting pixels for
```

Comment:
Normative requirement that no error is generated -> the SHALL is appropriate here?

Reply:
This merely defines what an error condition is, so I do not think a _SHALL_ is necessary. Up to you.

Reply:
See https://github.com/w3c/imsc-hrm/pull/60/commits/0df2807667eb2b875a9abf7820fa778ad156df01